meteo-notebook.ipynb

```diff
@@ -110,7 +110,7 @@
   },
```

Comment:
<p> </p><p>typo:  Boundary layer height increases en decreases later at night.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>en = and is corrected</p>